test/real_cases_periodic_test.rb

```diff
@@ -164,10 +165,28 @@ def test_fill_days_and_post_processing
       # checks performance on instance calling post_processing
       vrp = TestHelper.load_vrp(self, fixture_file: 'periodic_with_post_process')
       vrp.resolution_minimize_days_worked = true
-      result = OptimizerWrapper.wrapper_vrp('ortools', { services: { vrp: [:ortools] }}, vrp, nil)
+      found_uninserted_visits = 1000 # we start with 1000 unassigned visits
```

Comment:
why this initialization?

Reply:
This line should in fact be after line 173 : we are simulating phase one of periodic heuristic by setting @candidate_routes value.

The aim of this test is to test end_phase (refine_solution). 

To sum up : this instance has 3733 visits and after first stage we have 1000 still unassigned. We expect end_phase to plan some of those. 

In fact you are right, we do not need to initialize this variable, a simple comment would be enough.

Reply:
However, if I do not initialize this value here, I can not use it as I do in this test.

Reply:
Is it possible to deduce the value 1000 from the data ?

Reply:
We can deduce it from fill_days_and_post_processing_services_assignment.dump file but, still, that would be in stub function so not accepted.

Reply:
I can move data initialization before stub, and compute this variable from it. It makes less sense for me.